src/main/js/main_html5.js

```diff
@@ -508,6 +514,11 @@ require("../../../html5-common/js/utils/environment.js");
             playPromise.catch(_.bind(function(error) {
               if (error) {
                 OO.log("Play Promise Failure", error, error.name);
+                //PLAYER-3601: Workaround of an issue where play promises sometimes fail on iOS with Freewheel ads.
+                //We can ignore these as the Freewheel ad plugin will take care of these if they are indeed errors
+                if (OO.isIos && _video._fw_videoAdPlaying) {
```

Comment:
I've tried to figure out what is causing these play promise errors, but I couldn't find the root cause. The iOS play promise error messages are useless. I did find out we can safely ignore these as FW will render the ad just fine, so I've added this line here. I don't like it, but I'm not sure what else we can do in this spot.